spec/requests/api/tenant_quotas_spec.rb

```diff
@@ -0,0 +1,263 @@
+describe "tenant quotas API" do
+  let!(:root_tenant) { Tenant.seed }
```

Comment:
Do we need to add this? Looking at the tenant factory, the default parent is the root tenant, and I don't think this stands out as an important detail in these tests